app/views/transactions/_form.html.erb

```diff
@@ -0,0 +1,28 @@
+
+<%= form_with(model: [@category, @transaction], url: category_transactions_path(@category)) do |form| %>
+  <% if @transaction.errors.any? %>
+    <div id="error_explanation">
+      <h3><%= pluralize(@transaction.errors.count, "error") %> prohibited this transaction from being saved:</h3>
+
+      <ul>
+      <% @transaction.errors.full_messages.each do |message| %>
+        <li><%= message %></li>
+      <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div>
+    <%= form.label :name %>
+    <%= form.text_field :name %>
+  </div>
```

Comment:
- [ ] Dear @wolterbwambale Great job updating the splash page as suggested. But, just as mentioned by the previous reviewers we need to update your `transactions` view to match the original design.  

### Your design 
<img width="383" alt="Screenshot 2023-11-23 at 4 17 59 in the afternoon" src="https://github.com/wolterbwambale/Budget-app/assets/69077061/f3e49d33-7f47-42bd-8af0-1c77413ff1e6">

### Example for the expected design 

![](https://user-images.githubusercontent.com/58642949/285068106-7ad9e297-39cb-464e-aef3-623d4e7e4b0c.png)

Reply:
### [OPTIONAL]
- You are actually required to follow the [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) provided to you in the project requirements and not the one provided to you by the previous reviewer. The transaction and category pages should look like the image below
![image](https://github.com/wolterbwambale/Budget-app/assets/45460192/10d82508-9665-45f6-80c4-b53c7d9268fb)

In this project, you are required to follow the design guidelines which include:
  - Colors
  - Typography: font face, size and weight.
  - Layout: composition and space between elements.
  
That means, your design should closely resemble the original design for example the new category button should be the colour #5fb523, the text of the button should have a white border, and the listed transactions should have an image with the layout similar to the original design and so on. In short, the design should look almost exactly the same. If you get the time, please change the design provided in the project requirements since this is an optional review. For now, I will follow the design the 2 previous reviewers shared with you. Your design looks like this currently, compare it with the original design shared above and you will spot the differences.
![image](https://github.com/wolterbwambale/Budget-app/assets/45460192/07e1750c-67c4-42a0-99e4-cb95b4f9695d)
![image](https://github.com/wolterbwambale/Budget-app/assets/45460192/5aed52f2-fecf-4cae-9ad0-9b450a6d01a4)